cnd/src/swap_protocols/lnbtc.rs

```diff
@@ -0,0 +1,12 @@
+use crate::swap_protocols::{asset, identity};
+
+/// Data required to create a swap that involves a bitcoin on the lightning
+/// network.
+#[derive(Clone, Debug)]
+pub struct CreatedSwap {
+    // FIXME: Should this be `asset::bitcoin`?
```

Comment:
I think so yes.

Reply:
Went a step further and removed the `asset::Lightning` module all together, I should never have added it - it was an overdone abstraction.